File: app/src/main/java/pproject/teamjavis/javis/activity/LockActivity.java
package pproject.teamjavis.javis.activity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import java.util.ArrayList;
import java.util.List;

import pproject.teamjavis.javis.R;

public class LockActivity extends BaseActivity{

    private ListView list;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_lock);

        list = (ListView) findViewById(R.id.list);

        List<String> data = new ArrayList<>();

        ArrayAdapter<String> adp = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, data);

        list.setAdapter(adp);

        data.add("엄마");
        data.add("아빠");
        data.add("첫째");
        data.add("둘째");
        adp.notifyDataSetChanged();

    }
}

File: app/src/main/java/pproject/teamjavis/javis/activity/LockActivity.kt
/*
권한 설정 액티비티 클래스.

등록된 사용자의 권한 설정을 담당하는 클래스. 권한이 없는 행동은 명령 불가.
 */
package pproject.teamjavis.javis.activity

import android.os.Bundle
import kotlinx.android.synthetic.main.layout_topbar.*
import pproject.teamjavis.javis.R

class LockActivity: BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_lock)
        activityTitle.text = resources.getString(R.string.title_lock)

        backButton.setOnClickListener {
            finish()
        }
    }
}